dataloader/dataloader_utils.py

```diff
@@ -90,7 +90,7 @@ def should_download_file(file_lang: str, file_name: str) -> bool:
         return True
     if file_lang == LANG_SANSKRIT:
         return True
-    if file_lang == LANG_TIBETAN:
+    if file_lang == LANG_TIBETAN and "N" in file_name:
```

Comment:
This is not really needed but I understand this changes so often! I always make the mistake of pushing things like that too.